### DISCUSSION TITLE
Could not find a set of Noto fonts to display all missing characters

### DISCUSSION THREAD

**Discussion by AbhiRodi** [community/community#21442] [Other Feature Feedback, Questions, & Ideas]
<p>I get error on vs code in debug console when taping on button were icon are not showing properly and drawer are not showing properly.  I am not able to fix this issue, lastly i am here for getting solution.</p>
<p>after checking in chrome or any emulator it shows error(not showing icon.done and drawer properly</p>
<p>you and see the login button…but not shwing the icons</p>
<p>here drawer was not showing properly. when i install this application in my android phone then also this problem is happening.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/8/f8b0406e28aa795db68656265036188d249f8d70.jpeg" data-download-href="/uploads/short-url/zu01RV2XPjzt64NYbb1vmyWVXEY.jpeg?dl=1" title="WhatsApp Image 2021-08-29 at 11.22.27 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/88835985/181064563-062d4240-5fac-46d3-83b1-eb634abd674f.jpeg" alt="WhatsApp Image 2021-08-29 at 11.22.27 PM" data-base62-sha1="zu01RV2XPjzt64NYbb1vmyWVXEY" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/8/f8b0406e28aa795db68656265036188d249f8d70_2_690x388.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/f/8/f8b0406e28aa795db68656265036188d249f8d70_2_1035x582.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/8/f8b0406e28aa795db68656265036188d249f8d70.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/8/f8b0406e28aa795db68656265036188d249f8d70_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">WhatsApp Image 2021-08-29 at 11.22.27 PM</span><span class="informations">1280×720 146 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>i also download google noto fonts and successfully transfer to system fonts of laptop but still this problem is not going.</p>
<p>My whole week is wasted to solve this problem but not able to do that.</p>

**Comment by AbhiRodi** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/9/99777fe0828043327b8b70d25e612f44de7de1c9.jpeg" data-download-href="/uploads/short-url/lTD4VFthgniDakbpOH4mDb2YgwF.jpeg?dl=1" title="WhatsApp Image 2021-08-29 at 11.27.27 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/88835985/181064564-b73095af-2aac-4e2e-8ae7-e37a7af43600.jpeg" alt="WhatsApp Image 2021-08-29 at 11.27.27 PM" data-base62-sha1="lTD4VFthgniDakbpOH4mDb2YgwF" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/9/99777fe0828043327b8b70d25e612f44de7de1c9_2_690x388.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/9/9/99777fe0828043327b8b70d25e612f44de7de1c9_2_1035x582.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/9/99777fe0828043327b8b70d25e612f44de7de1c9.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/9/99777fe0828043327b8b70d25e612f44de7de1c9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">WhatsApp Image 2021-08-29 at 11.27.27 PM</span><span class="informations">1280×720 156 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mpboom** [comment]
<p>Is this issue possible related to your situation? <a href="https://github.com/flutter/flutter/issues/76093" class="inline-onebox" rel="noopener nofollow ugc">Could not find a set of Noto fonts to display all missing characters · Issue #76093 · flutter/flutter · GitHub</a></p>

**Comment by AbhiRodi** [comment]
<p>yes, what is solution of that issue</p>

**Comment by mpboom** [comment]
<p>For the exact details of that issue, I’d suggest reading the discussion in the issue. If the issue seems similar, but no clear solution is provided, <a href="https://github.com/flutter/flutter/issues/new/choose" rel="noopener nofollow ugc">try opening one</a>.</p>

**Comment by AbhiRodi** [comment]
<p>what is noto fonts…which giving the problems in showing icons and some small stuff. How to get noto fonts.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The homepage for the Noto fonts is here: <a href="https://www.google.com/get/noto/" class="inline-onebox">Google Noto Fonts</a></p>
<p>How to install them depends on your system (OS, desktop environment). Most Linux distributions have packages for them.</p>
